tests/network/hotplug.go

```diff
@@ -74,6 +75,26 @@ var _ = SIGDescribe("nic-hotplug", func() {
 		It("can be hotplugged a network interface", func() {
 			Expect(libnet.CheckInterface(vmi, vmIfaceName)).To(Succeed())
 		})
+
+		It("cannot hotplug multiple network interfaces for a q35 machine type by default", func() {
+			By("hotplugging the second interface")
+			const secondHotpluggedIfaceName = "iface2"
+			Expect(
+				kubevirt.Client().VirtualMachineInstance(vmi.GetNamespace()).AddInterface(
+					context.Background(),
+					vmi.GetName(),
+					addIfaceOptions(networkName, secondHotpluggedIfaceName),
+				),
+			).To(Succeed())
+			Eventually(func() []corev1.Event {
+				events, err := kubevirt.Client().CoreV1().Events(vmi.GetNamespace()).List(context.TODO(), metav1.ListOptions{})
```

Comment:
`context.TODO()` is reserved for cases when you expect this to be changed with a different context.
I do not think this is the case here.